app/controllers/merchants/items_controller.rb

```diff
@@ -2,5 +2,6 @@ class Merchants::ItemsController < ApplicationController
 
   def index
     @merchant = Merchant.find(params[:merchant_id])
+    @items = @merchant.items
```

Comment:
It looks like you don't use this in your view (you use @merchant.items directly in the view) so this could be removed